### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by anshul-kai** [chirag04/react-native-tooltip#18]
This is a great plugin but having it work only on IOS makes it not as useful. Any plans on making this work on Android as well?

Thanks!

**Comment by joonhocho** [comment]
+1

**Comment by chirag04** [comment]
Happy to accept a PR. That said there is https://github.com/chymtt/ReactNativeDropdownAndroid which can be a potential solution

**Comment by DonovanCharpin** [comment]
@chirag04 a dropdown is not the same as a tooltip, the behavior is not the same at all.
+1 for the android support.

**Comment by chirag04** [comment]
afaik there is no standard tooltip like component on android. I think they added it now in Android M/N. not sure.

**Comment by DonovanCharpin** [comment]
Yes sometime it's not simple to find the same behavior on ios and android platform.

On my side, I used this library https://github.com/exponentjs/react-native-action-sheet which create actionsheet on both android and ios and I can put my options inside.

**Comment by fungilation** [comment]
@DonovanCharpin that's a good "fallback" but not as good as a contextual, inline tooltip.
